src/extension.ts

```diff
@@ -21,10 +22,15 @@ export function activate(context: vscode.ExtensionContext) {
 
 		// Display a message box to the user
 		vscode.window.showInformationMessage('Welcome to the jcasc-plugin');
-		const schemaURL = vscode.workspace.getConfiguration().get('jcasc.schemaURL');
-		const userName = vscode.workspace.getConfiguration().get('jcasc.userName');
-		const userToken = vscode.workspace.getConfiguration().get('jcasc.userToken');
+		var schemaURL = vscode.workspace.getConfiguration().get('jcasc.schemaURL');
```

Comment:
why change this too var?

and why is this file all tabs =/, javascript is normally spaces